build/Gemfile

```diff
@@ -8,6 +8,7 @@ group :jekyll_plugins do
   gem "jekyll-feed", "0.17.0"
   gem "jekyll-sitemap", "1.4.0"
   gem "jekyll-tidy", "0.2.2"
+  gem 'jekyll-sass-converter', "3.0.0"
```

Comment:
this is way to get around the builtin libsass/sassc processor in favor of more modern implementations that support modern syntax (without this was unable to build a recent copy of Primer given the use of variables and various units)